config/peerpods/podvm/Dockerfile.podvm-builder

```diff
@@ -29,6 +27,9 @@ RUN if [[ -n "$CERT_RPM" ]] ; then \
 
 RUN git clone ${CAA_SRC} -b ${CAA_REF} /src/cloud-api-adaptor
 
+ADD rhel-guest-image-9.2-20230414.17.s390x.qcow2 /src/cloud-api-adaptor/src/cloud-api-adaptor/podvm/rhel-guest-image-9.2-20230414.17.s390x.qcow2
```

Comment:
Need to change : download the image while running the job than bundling it along the builder image